pkg/operations/kubernetesupgrade/upgradecluster.go

```diff
@@ -70,7 +70,7 @@ func (uc *UpgradeCluster) UpgradeCluster(subscriptionID uuid.UUID, resourceGroup
 	}
 
 	switch ucs.OrchestratorProfile.OrchestratorVersion {
```

Comment:
Also @rjtsdl 

> Another dimension is, upgrade so far is only handling scenario of kubernete 1.5.3 -> 1.6.2. 

Most important part of the upgrade operation is that it verifies the logic that upgrade from one version to another works. The fact that 1.5.3 will be discontinued doesn't mean that upgrade op code is not usable. It's a matter of changing the target version and then it will work.